### ISSUE TITLE
Overflow='hidden' not removed after Modal is closed  v0.8 

### ISSUE THREAD

**Issue by Hellycat** [chakra-ui/chakra-ui#2034]
# Bug report
Overflow='hidden' is not removed after Modal is closed

## To reproduce
1. Open Modal component
2. Click button or press ESC to close Modal
3. Scroll doesn't work

## Minimal reproduction
Demo: https://codesandbox.io/s/exciting-glitter-2f51x?file=/src/pages/index.js

## Expected behavior
 Scroll works

## System information
- macOS
- Browser Chrome
- Version of @chakra-ui/core: v 0.8.0
- Version of Next.js: 9.1.7

**Comment by ljosberinn** [comment]
Hi @Hellycat, thanks for reporting this but we're very close to releasing 1.0 and are no longer doing work on v0.8. Hope you understand!

**Comment by Hellycat** [comment]
Thank you @ljosberinn!
Is there any specific date when you expect to release 1.0? I haven't found any plan for that at docs

**Comment by ljosberinn** [comment]
_Soon (tm)_ - we've had 6 `.next` releases and additionally will release `.rc4` in the upcoming days so we're fairly close but can't give definite dates.

**Comment by Hellycat** [comment]
Looking forward to it :)

Thank you for a nice product

**Comment by ScholtenSeb** [comment]
If anyone else also has this issue, a temporary solution for me was to set the prop `blockScrollOnMount` to `false` on the modal.

```typescript
<Modal blockScrollOnMount={false}>
   ...
</Modal>
```

Looking forward to v1 official release :)

**Comment by spykr** [comment]
It seems like this issue was introduced in `v3.1.3` of  the [`body-scroll-lock`](https://github.com/willmcpo/body-scroll-lock) dependency, so you can fix it by pinning the version back to `v3.0.3`.

If you use Yarn you can achieve this by adding the following to your package.json file:
```
"resolutions": {
  "body-scroll-lock": "3.0.3"
}
```
